### ISSUE TITLE
SkipFiles doesn't seem to work

### ISSUE THREAD

**Issue by Spown** [hbenl/vscode-firefox-debug#25]
Followup of #20

I've tested the new ``SkipFiles`` option and sadly it doesn't seem to work at all. I've tryed ``"${workspaceRoot}/static/js/libs/**", "**/static/js/libs/**", "${workspaceRoot}/static/js/libs/jquery-1.11.2.min.js"`` and still jquery-1.11.2.min.js is opened in a new tab...

**Comment by Spown** [comment]
[/Spown/vsc_ff_debug_test_webapp/blob/c0c11dd4132ce226a65a48afa349bbef7c20ec2f/static/js/index.js#L5](test case)

**Comment by hbenl** [comment]
Damn, it's that same old problem with the debug protocol (as in #7) again: I can't tell Firefox that I want to blackbox the file until Firefox loaded and started to execute it - by the time my blackboxing message reaches Firefox it's too late, the breakpoint has already been hit.
But in this case (unlike #7) I think there is a way how to fix the problem. Stay tuned.

**Comment by hbenl** [comment]
OK, it should work now (at least it does with your test case on my machine).

**Comment by Spown** [comment]
nope, still opening the lib with any matching pattern I could have came up with... It makes one additional tap into the file where the breakpoint is before jumping to the lib, IINM.

**Comment by hbenl** [comment]
The minimatch library I use for the glob patterns didn't work as expected on Windows. Please try again now.

**Comment by Spown** [comment]
eyep, now i does seem to work! thank you!
